### ISSUE TITLE
Please, update the deprecated modules

### ISSUE THREAD

**Issue by rbatllet** [gulpjs/gulp#1806]
In Node 6.5.0, it shows these **SECURITY and DEPRECATED warning messages**:
```
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```

Outdated minimatch:
```
├─┬ gulp@3.9.1
│ └─┬ vinyl-fs@0.3.14
│   ├─┬ glob-stream@3.1.18
│   │ └── minimatch@2.0.10 
│   └─┬ glob-watcher@0.0.6
│     └─┬ gaze@0.5.2
│       └─┬ globule@0.1.0
│         └── minimatch@0.2.14 
```

Outdated Lodash:
```
├─┬ gulp@3.9.1
│ └─┬ vinyl-fs@0.3.14
│   └─┬ glob-watcher@0.0.6
│     └─┬ gaze@0.5.2
│       └─┬ globule@0.1.0
│         └── lodash@1.0.2 
```

Outdated graceful-fs:
```
├─┬ gulp@3.9.1
│ └─┬ vinyl-fs@0.3.14
│   ├─┬ glob-watcher@0.0.6
│   │ └─┬ gaze@0.5.2
│   │   └─┬ globule@0.1.0
│   │     └─┬ glob@3.1.21
│   │       └── graceful-fs@1.2.3 
│   └── graceful-fs@3.0.11 
```

**Comment by heikki** [comment]
https://github.com/gulpjs/gulp/issues?q=is%3Aissue+deprecated+is%3Aclosed

**Comment by phated** [comment]
Nope. Duplicate as @heikki mentioned.

**Comment by rbatllet** [comment]
In what year will it be solved? The outcoming 4.0 is still an alpha 2 !

**Comment by alexeyraspopov** [comment]
@rbatllet, are you willing to [help](https://github.com/gulpjs/gulp/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)?

**Comment by rbatllet** [comment]
Are you saying that you don't have enough staff to release the next final version?

**Comment by callumacrae** [comment]
gulp is an open source project and is maintained by contributors who volunteer their spare time to work on the project. It will be done when it is done. If you want it to be done quicker, you are very welcome to contribute some of your own time.

**Comment by rbatllet** [comment]
The Gulp 4.0 version was announced on August 25, 2014 (2 years ago!) -> https://twitter.com/gulpjs/status/504031119445336064

Unfortunately now, it is not actively developed. It seems it is dying slowly. Look at the graphs -> https://github.com/gulpjs/gulp/graphs/contributors?from=2016-01-1&to=2016-08-27&type=c

**Comment by callumacrae** [comment]
https://github.com/gulpjs/gulp/issues/1782

You are very welcome to contribute to the project.

**Comment by contra** [comment]
@rbatllet It isn't dying, but we're busy. We've responded to these tickets so many times - we have lives too.

I've had to move across the country 3 times since the tweet you linked - had my business unexpectedly fail (startups!), had relationships fail, deaths, travel, poverty, etc. - where the fuck am I supposed to find time to work on this while I'm trying to pay rent and put food in my stomach? The attitude people have towards open source authors is disgusting - I don't owe you anything, nobody does. Unless you've done something for the open source community (hint: you haven't!) then you aren't entitled to shit. If you want something to happen then help make it happen.
